src/components/common/Notifications/index.tsx

```diff
@@ -50,6 +51,7 @@ const Toast = ({
   link,
   onClose,
   id,
+  autoHide = true,
```

Comment:
Nit: I would flip this to something like `persist` instead as autohiding is a default of ours.

Reply:
Good idea! Will change it to `persist`